test/io/readfile/Reading_from_pipe.chpl

```diff
@@ -0,0 +1,14 @@
+use Subprocess;
+use IO;
+var sub = spawn(["echo","How are you?"],stdout=PIPE);
```

Comment:
How about running `cat f2.txt` or `cat moby.txt` (if you create that link to moby.txt)?
It seems like it would be nice to test a longer file than what you have here but it could be 2 of them.